.github/workflows/nf-test.yml

```
@@ -61,7 +62,9 @@ jobs:
     needs: [nf-test-changes]
     if: ${{ needs.nf-test-changes.outputs.total_shards != '0' }}
     runs-on: # use self-hosted runners
-      - runs-on=${{ github.run_id }}-nf-test/runner=4cpu-linux-x64/disk=large
+      - runs-on=${{ github.run_id }}-nf-test
+      - runner=4cpu-linux-x64
+      - disk=large
```

```suggestion
      - volume=80gb
```
switch to new runsOn syntax https://github.com/runs-on/runs-on/releases/tag/v2.9.0